deploy/README.md

```diff
@@ -125,6 +125,16 @@ REQUIRED_NUMBER_OF_VALIDATORS=1
 # the Foreign network to confirm that the finalized agreement was transferred
 # correctly to the Foreign network.
 VALIDATORS="0x 0x 0x"
+
+# The flag defining whether to use ERC677BridgeTokenRewardable contract instead of
+# ERC677BridgeToken.
+DEPLOY_REWARDABLE_TOKEN=false
+# The address of ValidatorSet contract used by ERC677BridgeTokenRewardable contract.
+# Makes sense only when DEPLOY_REWARDABLE_TOKEN=true
+VALIDATOR_SET_ADDRESS=0x
```

Comment:
hmm... wait a minute: is this bridge validators or network validators? If they are just for network let's rename the parameter to `DPOS_VALIDATOR_SET_ADDRESS`.

Reply:
This is the address of [`ValidatorSet` contract](https://github.com/poanetwork/pos-contracts/blob/master/contracts/ValidatorSetAuRa.sol) which contains network validators. Ok, I've renamed the param: https://github.com/poanetwork/poa-bridge-contracts/pull/129/commits/47f766b6c90759281abffee38e0c0834417ed727